src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask.xml

```diff
@@ -475,6 +475,24 @@
             </summary>
             <param name="result">The result.</param>
         </member>
+        <member name="M:Microsoft.Azure.WebJobs.Extensions.DurableTask.RemoteOrchestratorContext.ThrowIfPlatformLevelException(DurableTask.Core.FailureDetails)">
```

Comment:
Any idea why this is showing up in our public documentation? I thought these APIs were internal?

Reply:
My guess is that since I added method-level comments to the private method, they're made it to the xml. I think this is probably just to power the intellisense? Not sure, just a guess.

Reply:
It's not just intellisense, but also the public API docs that we publish, like this: https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.webjobs.extensions.durabletask?view=azure-dotnet. If it's not meant to be published, then we should remove it from this xml file. Might be worth doing a bit of research as when when/why non-public APIs are getting included in this file.

Reply:
Seems that adding the method-level comments was the culprit.
I removed them (moved them to the method body), and the xml change was undone: https://github.com/Azure/azure-functions-durable-extension/pull/2922/commits/c92726846c351aeb341c80fd3f0705e6b93653c4